### PR TITLE
feat: support qovery-container-names input

### PR DESCRIPTION
**Pull request by Thiht**
Using names is more convenient than IDs for us

### CODE REVIEW

**Comment by benjaminch** [comment]
Hello @Thiht !
Sorry I've missed that, we will have a look.
Cheers

File: github-action/main.go [hunk 1]
```diff
@@ -97,6 +98,26 @@ func getApplicationIds(qoveryAPIClient pkg.QoveryAPIClient, envId string, id *st
...
 }
 
+func getContainerIds(qoveryAPIClient pkg.QoveryAPIClient, envId string, id *string, name *string) (string, error) {
+	if id != nil && *id != "" {
+		return sanitizeInputIDsList(*id), nil
+	}
+
+	if name != nil && *name != "" {
+		var ids []string
+		for _, sName := range strings.Split(*name, ",") {
```
Comment (benjaminch) [inline github-action/main.go:108]: Nit: I know other similar functions doesn't have this in place but I would make sure to trim names / IDs generally speaking.
We have been hit by a similar case in last couple weeks, and might happen here as well.

CF PR fixing it => https://github.com/Qovery/qovery-github-actions/pull/11

**Review by benjaminch — COMMENTED** [review]
Looks good to me, just a nitpick comment to handle if you want ;)
Let me know if you need any help.

Again thanks a lot for your contribution here, very appreciated ;)

File: github-action/main.go (hunk 1 above)
Comment (Thiht) [reply github-action/main.go]: Updated :)

**Review by benjaminch — APPROVED** [review]
LGTM :)

**Comment by benjaminch** [comment]
@Thiht  Here you go, thanks again for your contribution :)
https://github.com/Qovery/qovery-github-actions/releases/tag/v0.15.7

**Comment by Thiht** [comment]
Happy to contribute! thanks for the release

**Pull request merged**